.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -32,6 +32,35 @@ body:
       placeholder: Steps to reproduce, e.g. "Go to [Matomo-URL] > Click on Visitors > Click Overview"
     validations:
       required: true
+  - type: input
+    id: customer-website-name
+    attributes:
+      label: Customer and Website Name
+      description: Customer Name and Website Name
+    validations:
+      required: true
+  - type: dropdown
+    id: installation
+    attributes:
+      label: On Premise or Cloud
+      multiple: false
+      options:
+        - On Premise
+        - Cloud
+    validations:
+      required: true
+  - type: dropdown
+    id: priority
+    attributes:
+      label: Priority
+      multiple: false
+      options:
+        - 1 - Critical
+        - 2 - High
+        - 3 - Medium
+        - 4 - Low
+    validations:
+      required: false
```

Comment:
Do you want the reporter to set a priority? What additional value do you hope to get from having that information being provided from a single user?

Reply:
@sgiehl Yes, the intent is to get an initial gauge from the end user on how important and urgently the bug needs to be fixed. We will still follow our internal triage process to ensure the right internal priority is set.

Reply:
If reporters have the option to bump the priority on their issues, the majority of them will put high or critical. Let's see, we can always remove it again.